guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -42,6 +42,16 @@ For information on backups, see {AdministeringDocURL}Backing_Up_Server_and_Proxy
 
 ifdef::katello,satellite[]
 +
+. Confirm the certificate tar file name on existing {smart-proxy-context} server:
++
+[options="nowrap" subs="attributes"]
+----
+# grep tar_file /etc/foreman-installer/scenarios.d/capsule-answers.yaml
```

Comment:
It's most likely named "katello-answers" or "foreman-answers" on other scenarios; so we have to use an attribute.

Reply:
As an installer maintainer I really don't like grepping in the answers file in our instructions. Answer files should be considered an internal installer detail.

IMHO you should always pass in the value if you want to guarantee it's set to a certain value, rather than making sure files are in places that the answers dictate.

So in practice:
* Generate the tar file on the `{Project} server`
* Copy it over to `{smart-proxy-context} server`
* Run the installer with `--tar-file /path/to/tar` on `{smart-proxy-context} server`.

Reply:
Not sure what changes are required in the doc.

Reply:
Please drop this step since it's not needed. All the way at the bottom the installer command is:
```
# foreman-installer --certs-tar-file /root/myproxy.example.com-certs.tar \
```

That means the actual answer for tar_file in `*-answers.yaml` is set to `/root/myproxy.example.com-certs.tar`, regardless of what the value was. Checking it at this point can only create confusion.

Reply:
Dropped this step and added a new step after the discussion with original issue created. Thanks :)